src/i18n/locales/es.json

```diff
@@ -3,6 +3,7 @@
   "blog.description": "The latest Node.js news, case studies, tutorials, and resources.",
   "blog.authors.list.title.by": "by",
   "blog.authors.list.title.and": "and",
+  "components.author.githubLinkLabel": "{username} Github - opens in new tab",
```

Comment:
Usually, we add new entries at the bottom, so that we know they're additions. Also reduces the changes of conflicts 👀

Reply:
Ah ok, I just assumed it was in alphabetical order 😆